test_scripts/Polices/appID_Management/TS01.lua

```diff
@@ -0,0 +1,83 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary: 
+--     [OnAppInterfaceUnregistered] "APP_UNAUTHORIZED" in case of failed nickname validation after updated policies
+-- Description: 
+--     In case the application with "appName" is successfully registered to SDL
+--     and the updated policies do not have this "appName" in this app's specific policies
+--     (= "nicknames" filed in "<appID>" section of "app_policies")
+--     SDL must send OnAppInterfaceUnregistered (APP_UNAUTHORIZED) to such application.
+-- 
+-- Test:
```

Comment:
@dboltovskyi According to template lines 10-18 should be removed.
You may create additional file for Testing coverage.